src/common/adapters/jira.test.js

```diff
@@ -1,26 +1,30 @@
+import { JSDOM } from 'jsdom';
+
 import client from '../client';
 import scan from './jira';
 
-const selectedIssue = 'RC-654';
-const issueType = 'Story';
-const title = 'A quick summary of the ticket';
+jest.mock('../client', () => jest.fn());
+
+const key = 'RC-654';
 
 const response = {
   id: '10959',
-  key: selectedIssue,
   fields: {
-    issuetype: {
-      name: issueType,
-    },
-    summary: title,
+    issuetype: { name: 'Story' },
+    summary: 'A quick summary of the ticket',
   },
+  key,
 };
 
-jest.mock('../client', () => jest.fn());
+const ticket = {
+  id: response.key,
+  title: response.fields.summary,
+  type: response.fields.issuetype.name.toLowerCase(),
+};
 
 describe('jira adapter', () => {
-  function loc(host, pathname = '', params = null) {
-    const searchParams = new URLSearchParams(params || {});
+  function loc(host, pathname = '', params = {}) {
```

Comment:
I set the default value of `params` to `null` because otherwise in the ternary operator below it is always true, I mean:

```js
const href = params ? `https://...?params` : `https://...`;
```

I see these alternatives:
- we can remove the ternary operator at all, there will be always the `?` at the end of the host, but doesn't really matter
- we can check that its keys length is more than 0 `Object.keys(params).length > 0 ? ... : ...`
- we can rollback these changes to the previous implementation

I don't have a strong opinion about that.

Reply:
I see now. I missed that yesterday evening. Thanks, I'll revert it.

Reply:
Done 👍